### PR TITLE
Add an option to expose Prometheus metrics via http/s in the forwarder-vpp

### PR DESCRIPTION
**Pull request by bszirtes**
## Description

Added a section in the main for initializing Prometheus and extended the config with the required variables.

## Issue link

https://github.com/networkservicemesh/sdk/issues/1652

## How Has This Been Tested?

- [ ] Added unit testing to cover
- [X] Tested manually
- [ ] Tested by integration testing
- [ ] Have not tested

## Types of changes

- [ ] Bug fix
- [X] New functionality
- [ ] Documentation
- [ ] Refactoring
- [ ] CI

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -149,6 +149,52 @@ func main() {
...
+	if opentelemetry.IsPrometheusEnabled() {
+		collectorAddress := cfg.OpenTelemetryEndpoint
+		spanExporter := opentelemetry.InitSpanExporter(ctx, collectorAddress)
+		metricExporter := opentelemetry.InitPrometheusMetricExporter(ctx)
+		o := opentelemetry.Init(ctx, spanExporter, metricExporter, cfg.Name)
+		defer func() {
+			if err = o.Close(); err != nil {
+				log.FromContext(ctx).Error(err.Error())
+			}
+		}()
```
Comment (denis-tingaikin) [inline main.go:164]: Curious. Why do we need to use opentemeltry in the prometheus branch?

**Comment by bszirtes** [comment]
Hi @denis-tingaikin,

I have rebased the branch to the latest sdk.

File: internal/config/config.go [hunk 2]
```diff
@@ -36,19 +36,22 @@ import (
...
+	LogLevel                      string            `default:"INFO" desc:"Log level" split_words:"true"`
+	DialTimeout                   time.Duration     `default:"750ms" desc:"Timeout for the dial the next endpoint" split_words:"true"`
+	OpenTelemetryEndpoint         string            `default:"otel-collector.observability.svc.cluster.local:4317" desc:"OpenTelemetry Collector Endpoint" split_words:"true"`
+	MetricsExportInterval         time.Duration     `default:"10s" desc:"interval between mertics exports" split_words:"true"`
+	PprofEnabled                  bool              `default:"false" desc:"is pprof enabled" split_words:"true"`
+	PprofListenOn                 string            `default:"localhost:6060" desc:"pprof URL to ListenAndServe" split_words:"true"`
+	PrometheusIP                  string            `default:"" desc:"Prometheus IP address" split_words:"true"`
+	PrometheusPort                uint16            `default:"8081" desc:"Prometheus port to use" split_words:"true"`
```
Comment (denis-tingaikin) [inline internal/config/config.go:53]: Can be combined to PrometheusListenOn by analogy with `ListenOn`, `PprofListenOn`.

**Review by denis-tingaikin — COMMENTED** [review]
Added one comment, in general looks good.

**Comment by bszirtes** [comment]
Hi @denis-tingaikin,

Thanks! I've made the modification as you recommended.

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**